### ISSUE TITLE
Improve accessibility social buttons, move onClick from li element to link element a

### ISSUE THREAD

**Issue by georgel-pop-lr** [liferay/clay#4460]
Description: A client is complying about an accessibility issue we have inside Asset publisher when activating the share on social media button. The problem they complain about is that we add the onClick action on the li element, and it should always be on a button or link, which are accessible by keyboard action. As a result is breaking: https://www.w3.org/WAI/WCAG21/Techniques/client-side-script/SCR35.html

Steps:
1. Add an asset publisher to a page, be sure you have some web contents created
2. Open configuration of asset publisher and add some SOCIAL BOOKMARKS (for example Facebook, Twitter and LinkedIn)
3. Close the configuration and inspect the Facebook button (see the HTML DOM)

Actual: The onClick is added to the li element and it should be added to the a element

Expected: The onClick is added to the a element, like this we could improve accessibility

For more details, check: https://issues.liferay.com/browse/LPS-143101

**Comment by pat270** [comment]
Going to close this issue looks like it was merged and released. Let me know if I'm wrong.
